src/components/AllRequests.jsx

```diff
@@ -14,7 +14,7 @@ const AllRequests = () => {
 
   useEffect(() => {
     getRequests(dispatch, activeCategory, coords);
-  }, [dispatch, activeCategory]);
+  }, [dispatch, activeCategory, longitude]);
```

Comment:
Why longitude though, and not coords?

Reply:
Coords is not a redux state, will it work ? I didnt check